pom.xml

```diff
@@ -58,6 +58,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Please create 2 separate pull requests - one for Unit tests, and another for Dao optimization